ghost/tinybird-dedicated/scripts/append_fixtures.sh

```
@@ -0,0 +1,21 @@
+
+#!/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)